test/components/common/stat/Stat.test.js

```diff
@@ -2564,5 +2593,40 @@ describe('Stat', () => {
 
       expect(wrapper.state().inputSuffix.value).to.equal(600);
     });
+
+    it('should call `propagateInputChange` after setting state', () => {
```

Comment:
This is the same description as one added just above. Both are testing various state setting, but maybe the descriptions could be differentiated just a little?

Reply:
I've differentiated them.